src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/touchstoneRepository/GetScenarioTests.kt

```diff
@@ -90,9 +90,11 @@ class GetScenarioTests : TouchstoneRepositoryTests()
 
     private fun checkScenarioIsAsExpected(result: ScenarioAndCoverageSets, extraTouchstones: List<String> = emptyList())
     {
-        assertThat(result.scenario).isEqualTo(Scenario(
-                "yf-1", "Yellow Fever 1", "YF", listOf(touchstoneId) + extraTouchstones
-        ))
+        assertThat(result.scenario.description).isEqualTo("Yellow Fever 1")
+        assertThat(result.scenario.id).isEqualTo("yf-1")
+        assertThat(result.scenario.disease).isEqualTo("YF")
+        assertThat(result.scenario.touchstones).hasSameElementsAs(listOf(touchstoneId) + extraTouchstones)
```

Comment:
Making this test invariant under ordering of the touchstones, as I don't think we care about this (ordering by touchstone id seems reasonable, but happens to flip the order in some tests here)